src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,127 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classes from 'classnames';
 import './FindMovie.scss';
 
+import { getMovie } from '../../api/api';
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+
+export const FindMovie = ({ addMovie }) => {
+  const [movieTitle, updateMovieTitle] = useState('');
+  const [loadMovie, createMovie] = useState(null);
+  const [error, toggleError] = useState(false);
+  const [preview, setPreview] = useState(false);
+  const [errorAddMovie, setMovie] = useState(false);
+
+  const findMovie = async() => {
+    const movie = await getMovie(movieTitle);
+
+    if (movie.Response === 'False') {
+      toggleError(true);
+      setPreview(false);
+
+      return;
+    }
```

Comment:
if we already have a movie in the state, after a false response better to delete it, because at this moment I can add a movie from the state if I tried to find another move, and got a false response